documentation/modules/ROOT/pages/05-quarkus-panache.adoc

```diff
@@ -6,8 +6,8 @@ include::_attributes.adoc[]
 
 At the end of this chapter you will be able to :
 
-* What is Panache ?
-* How to build a Hibernate application using Panache?
+* [x] What is Panache ?
+* [x] How to build a Hibernate application using Panache?
```

Comment:
shall we do the same in all the chapters for consistency?

Reply:
I have been already updating in all other PRs we have been doing so everywhere is in this way now.